frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js

```diff
@@ -171,22 +171,19 @@ export async function requestFromStore(
     const errorMessage = `Requesting data from layer "${
       layerInfo.name
     }" failed. Some rendered areas might remain empty. Retrying...`;
-    let detailedError =
+    const detailedError =
       errorResponse.status != null
-        ? `Status code ${errorResponse.status} - "${errorResponse.statusText}".`
+        ? `Status code ${errorResponse.status} - "${errorResponse.statusText}" - URL: ${
+            errorResponse.url
+          }.`
         : errorResponse.message;
 
-    detailedError += `(URL: ${dataUrl})`;
```

Comment:
If the error is not a response with a status code, the requests lib already appends the URL which is why I moved this from here to the conditional above.